flow/Makefile

```diff
@@ -469,6 +469,16 @@ do-$(1):
 	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/$(3).tcl -metrics $(LOG_DIR)/$(1).json) 2>&1 | tee $(LOG_DIR)/$(1).log
 endef
 
+define do-copy
```

Comment:
do-step equivalent for steps that simply copy files

Reply:
Please add defines that contain `-` to the EXCLUDED_VARS list in `flow/util/generate-vars.sh` as they can cause problems. If these `defines` should be included in the make issue, please use an underscore instead in the naming.  See #1240 for reference.